test/plugins/astro.test.ts

```diff
@@ -8,11 +8,13 @@ import baseCounters from '../helpers/baseCounters.js';
 const cwd = resolve('fixtures/plugins/astro');
 
 test('Find dependencies in Astro configuration', async () => {
-  const { counters } = await main({
+  const { issues, counters } = await main({
     ...baseArguments,
     cwd,
   });
 
+  assert(!issues.devDependencies['package.json']?.['@astrojs/check']);
```

Comment:
Many tests use `counters` to assert that there are no issues. So we don't need to assert the issues that are not there (you could verify this by removing either `@astrojs/check` from the dependencies or remove your implementation).

The conditional `?.` here is also a little bit confusing, does the test succeed or fail with or without the `['package.json']` key?

Reply:
Thanks! Then, I will remove the assertion.

For the condition, I intended the test to succeed if the `package.json` key does not exist or exists but `@astrojs/check` does not exist.